### ISSUE TITLE
ML example

### ISSUE THREAD

**Issue by saratheruvaram** [saratheruvaram/siva#2]
Sure! Here's an example of a simple machine learning program in Python using the scikit-learn library:

```python
# Importing the necessary libraries
from sklearn import datasets
from sklearn.model_selection import train_test_split
from sklearn.neighbors import KNeighborsClassifier
from sklearn.metrics import accuracy_score

# Loading the iris dataset
iris = datasets.load_iris()
X = iris.data
y = iris.target

# Splitting the dataset into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Creating the KNN classifier
knn = KNeighborsClassifier(n_neighbors=3)

# Training the classifier
knn.fit(X_train, y_train)

# Making predictions on the testing set
y_pred = knn.predict(X_test)

# Calculating the accuracy of the classifier
accuracy = accuracy_score(y_test, y_pred)
print("Accuracy:", accuracy)
```

This program uses the iris dataset from scikit-learn, splits it into training and testing sets, creates a KNN classifier, trains the classifier on the training set, makes predictions on the testing set, and calculates the accuracy of the classifier.
